docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -0,0 +1,291 @@
+= Instance configuration
+
+The Instance is the top-level openDAQ(TM) object, serving as a container for the openDAQ context and the base module manager.
+
+**Related articles:**
+
+  * xref:background_info:opendaq_architecture.adoc#instance[Architecture - Instance]
+
+== Creating the default openDAQ(TM) Instance
+Upon creation, it generates a Client device, which is a default device implementation capable of loading any function blocks present in the module manager's search path. If the native openDAQ client module is loaded, the Client device can connect to any TMS-enabled device using the `addDevice` function. The Client is set as the root device when the instance is created.
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+#include <iostream>
+
+int main(int argc, const char* argv[])
+{
+    daq::InstancePtr instance = daq::Instance();
+    daq::DevicePtr device = instance.addDevice("daq.opcua://127.0.0.1/");
+
+    return 0;
```

Comment:
This code sample is probably redundant for the scope of this guide - it should more be focused on the builder.